lib/src/git.rs

```diff
@@ -1293,6 +1293,8 @@ pub enum GitRemoteManagementError {
     NoSuchRemote(String),
     #[error("Git remote named '{0}' already exists")]
     RemoteAlreadyExists(String),
+    #[error("Git remotes with slashes are invalid: '{0}'")]
```

Comment:
nit: "invalid" sounds too strong. Maybe we can say something like "cannot be used", "is not compatible with jj", etc.?

btw, I would remove quotes from `'{0}'` if the parameter is separated by `: `.